### DISCUSSION TITLE
Canal Blau [ES]

### DISCUSSION THREAD

**Discussion by Kyngo** [iptv-org/community#389] [Show and tell]
### Channel ID

CanalBlau.es

### Stream URL

https://dh10totrr51mr.cloudfront.net/out/v1/ecf7401369af4edca3ce6453ca830fd0/index_2.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://dh10totrr51mr.cloudfront.net/out/v1/ecf7401369af4edca3ce6453ca830fd0/index_2.m3u8 (offline)
<img width="674" alt="image" src="https://user-images.githubusercontent.com/7253922/232257946-6fb1a8c9-518c-47f7-a0b9-c44f312f08eb.png">
